### ISSUE TITLE
Multiple displays problem

### ISSUE THREAD

**Issue by pit00** [Pxper/Pxper#2]
The guidelines only appear on the primary display.

**Comment by DmitrySikorsky** [comment]
Yes, I know about this issue. Will fix it if have time, this is not very popular project. I will be happy if you can help me with this bug.

**Comment by pit00** [comment]
No problem! I'll take a look at the code (anytime soon).
